natives.json

```diff
@@ -85494,15 +85494,15 @@
 			"build": "1207"
 		},
 		"0xA079300AF757FB1A": {
-			"name": "_0xA079300AF757FB1A",
+			"name": "_GET_VEHICLE_LIGHT_PROPSETS",
```

Comment:
PROPSET*S* implies it can return multiple prop sets but the return type is just ``PropSet``. Typo?